bundles/binding/org.openhab.binding.daikin/README.md

```diff
@@ -1,37 +1,55 @@
-Documentation of the Daikin binding bundle.
+# Daikin Binding
 
-## Introduction
+The openHAB Daikin binding allows monitoring and control of a Daikin air
+conditioner/heat pump.
 
-This binding allows you to monitor and control a Daikin air conditioner/heat pump via openHAB. In order for this to work a [KKRP01A](http://www.onlinecontroller.eu/) Online Controller must be installed in your Daikin unit and connected to your LAN. 
+<!-- MarkdownTOC depth=1 -->
+
+- [Prerequisites](#prerequisites)
+- [Binding Configuration](#binding-configuration)
+- [Item Configuration](#item-configuration)
+- [Examples](#examples)
+
+<!-- /MarkdownTOC -->
```

Comment:
I think tables of contents are only called for when there is a lot of content you would otherwise have to scroll and scroll through.